packages/protocol/contracts/L1/libs/LibVerifying.sol

```diff
@@ -90,11 +90,16 @@ library LibVerifying {
                 || config.blockMaxTxListBytes > 128 * 1024 // calldata up to 128K
                 || config.livenessBond == 0 || config.ethDepositRingBufferSize <= 1
                 || config.ethDepositMinCountPerBlock == 0
+            // Audit recommendation, and gas tested. Processing 32 deposits (as initially set in
+                // TaikoL1.sol) costs 72.502 gas. So a safe bet to set max cap around 100K (cca 45)
+                // as average propose block is somewhere 170-200K depending on the tier). It is
+                // anyway set by Taiko Labs.
+                || config.ethDepositMaxCountPerBlock > 45
```

Comment:
Seems like on Ethereum the limit is 16: 
- https://eips.ethereum.org/EIPS/eip-4895#why-no-gas-costs-for-the-withdrawal-type
- https://benjaminion.xyz/eth2-annotated-spec/phase0/beacon-chain/#max-operations-per-block

Because these deposits are actually pretty expensive as they require L1 work I don't think we have to follow this same value, but 16 would be the very conservative value.

Reply:
I make it max 32 then.